client/src/components/PageLayout/Header.tsx

```diff
@@ -69,32 +68,42 @@ export const Header: React.FC = () => {
                 >
                   Menu
                 </MenuButton>
-                <MenuList paddingBlock={0}>
+                <MenuList paddingBlock={0} as="ul">
                   <Flex
                     flexDirection="column"
                     fontWeight="600"
                     borderRadius="5px"
                   >
-                    <NextLink passHref href="/chapters">
-                      <MenuItem as="a">Chapters</MenuItem>
-                    </NextLink>
+                    <MenuItem as="li" cursor="pointer">
+                      <Link width="100%" href="/chapters">
+                        Chapters
+                      </Link>
+                    </MenuItem>
```

Comment:
I have wrapped the link around the text, because I think this way the screen readers read it as `link to chapter`.

While previously, they would have read `link link to chapter`.

Reply:
fixed